libraries/botbuilder-spell-check-middleware/package.json

```diff
@@ -37,6 +37,12 @@
     "azure-cognitiveservices-spellcheck": "^1.0.1",
     "botbuilder": "^4.1.3",
     "restify": "^7.2.2",
+    "typescript": "^3.2.2",
```

Comment:
Should Typescript not be a dev-dependency?

Reply:
Changed.